README.md

```diff
@@ -37,10 +37,12 @@ Setup and/or run CBench and/or OpenDaylight.
 OPTIONS:
     -h Show this message
     -v Output verbose debug info
-    -c Install CBench
```

Comment:
I suggest not removing the `-c` flag and instead just adding `-u` and `-f` flags to mark which OS-specific logic we should exercise. So, to install CBench on Fedora you'd still use `./wcbench.sh -c` (defaulting to what we currently support to maintain API backwards compatability), on Ubuntu you'd use `./wcbench.sh -uc`.